server/authstatus/service.go

```
@@ -0,0 +1,344 @@
+		return usagePayload{}, err
+	}
+	request.Header.Set("Authorization", authorization)
+	request.Header.Set("User-Agent", "builder/dev")
+	if accountID := strings.TrimSpace(state.Method.OAuth.AccountID); accountID != "" {
+		request.Header.Set("ChatGPT-Account-Id", accountID)
+	}
+	response, err := (&http.Client{Timeout: 10 * time.Second}).Do(request)
+	if err != nil {
+		return usagePayload{}, err
+	}
+	defer response.Body.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `response.Body.Close` is not checked

(errcheck)